### PR TITLE
feat(looker): ensure explore name matches looker's display name

### PR DESCRIPTION
**Pull request by shirshanka**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/looker_common.py [hunk 1]
```diff
@@ -462,6 +462,11 @@ def _get_fields_and_primary_keys(
                 primary_keys.append(schema_field.fieldPath)
         return fields, primary_keys
 
+    @staticmethod
+    def _display_name(name: str) -> str:
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/looker_common.py:466]: Do we usually use underscore at the beginning of "public" method names?

File: metadata-ingestion/tests/integration/looker/golden_test_allow_ingest.json [hunk 2]
```diff
@@ -72,7 +74,7 @@
                             "looker.explore.file": "test_source_file.lkml"
                         },
                         "externalUrl": "https://looker.company.com/explore/data/my_view",
-                        "name": "my_view",
+                        "name": "My Explore View",
```
Comment (jjoyce0510) [inline metadata-ingestion/tests/integration/looker/golden_test_allow_ingest.json:77]: Niceeee!!

**Review by jjoyce0510 — APPROVED** [review]
Overall LGTM

File: metadata-ingestion/src/datahub/ingestion/source/looker_common.py (hunk 1 above)
Comment (shirshanka) [reply metadata-ingestion/src/datahub/ingestion/source/looker_common.py:466]: did it this way to align with the other methods ... will come back and clean up this class later.

**Pull request merged**